### PR TITLE
Calendar add task solution

### PR DESCRIPTION
**Pull request by a-kovalov**
-[DEMO LINK](https://a-kovalov.github.io/layout_calendar/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__field">
```
Comment (DarkMistyRoom) [inline src/index.html]: why do you need this additional container? you can to the same via `calendar`

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  @for $monthLength from 28 through 31 {
+    &--month-length--#{$monthLength} {
+      .calendar__day:nth-child(n + #{$monthLength + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  $days: "mon" 0, "tue" 1, "wed" 2, "thu" 3, "fri" 4, "sat" 5, "sun" 6;
```
Comment (DarkMistyRoom) [inline src/styles/blocks/calendar.scss]: move it to a variables file

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+    @include square(100px);
+    background-color: #eee;
+    border: 1px solid black;
+    transition: background-color 0.5s, transform 0.5s;
```
Comment (DarkMistyRoom) [inline src/styles/blocks/calendar.scss]: move transition time to a variable

File: src/styles/blocks/calendar.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+    &:hover {
+      transform: translate(0, -20px);
+      cursor: pointer;
```
Comment (DarkMistyRoom) [inline src/styles/blocks/calendar.scss]: move `cursor: pointer;` to the main selector

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Good job, just a few fixes are needed:

please center the calendar
image

**Comment by a-kovalov** [comment]
- [DEMO LINK](https://a-kovalov.github.io/layout_calendar/) - fixed version uploaded

**Comment by a-kovalov** [comment]
Please ignore _Calendar fixed version_, in **_Calendar fixed v2_** - removed not need styles that left in _Calendar fixed._
- [DEMO LINK](https://a-kovalov.github.io/layout_calendar/)

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Oops! Something went wrong when I changed the modifier
image

**Comment by a-kovalov** [comment]
- [DEMO LINK](https://a-kovalov.github.io/layout_calendar/)
My mistake, justify-content fixed, all good must be now - checked before deploying xD
Thank you for your patience and help and sorry for spam

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Pull request closed without merging**